mypyc/primitives/dict_ops.py

```diff
@@ -142,6 +142,14 @@
     c_function_name='CPyDict_ItemsView',
     error_kind=ERR_MAGIC)
 
+# dict.clear()
+method_op(
+    name='clear',
+    arg_types=[dict_rprimitive],
+    return_type=void_rtype,
+    c_function_name='CPyDict_Clear',
+    error_kind=ERR_NEVER)
```

Comment:
Use `ERR_MAGIC` here since an overridden method might raise an exception.

Reply:
I'm a little confused about this one. If I change the `error_kind` to `ERR_MAGIC`, I get this error:
```
assert not op.is_void, "void op generating errors?"
```
Due to the fact that the return type is `void_rtype`. However, what should I change this to? `CPyDict_Clear` uses internally `PyDict_Clear` which has a `void` return type itself.

Reply:
Ah one option is t to use `bit_rprimitive` as the return type (`char` in the C function) and `ERR_FALSE` as the error kind. Then unconditionally return 1 when there can be no error. 0 signnals an exception being raised.

Reply:
@JukkaL I have added `case testDictMethods` to be used for testing `copy`, `clear` and any other dict methods. Let me know if it looks good to you.